includes/Modules/Analytics_4.php

```diff
@@ -242,6 +242,10 @@ protected function get_datapoint_definitions() {
 			'GET:accounts'               => array( 'service' => 'analyticsadmin' ),
 			'GET:container-lookup'       => array( 'service' => 'tagmanager' ),
 			'GET:container-destinations' => array( 'service' => 'tagmanager' ),
+			'GET:conversion-events'      => array(
+				'service'   => 'analyticsadmin',
+				'shareable' => Feature_Flags::enabled( 'dashboardSharing' ),
```

Comment:
Technically this flag is rolled out to 100% now but we should probably remove it in its respective issue.